group2.html

```diff
@@ -127,6 +127,7 @@
         </label>
       </div>
       <button type="submit">Submit</button>
+      <button type="button" id="reset">Reset</button>
```

Comment:
Good work on the localstorage! Small detail, I don't think it makes much sense to put the button here in the HTML (a form usually only has one submit button). It would probably make more sense to move it to the bottom of the HTML and have it say "reset data" (a bit more descriptive for a user)